diff --git a/storage-proofs/core/src/parameter_cache.rs b/storage-proofs/core/src/parameter_cache.rs
index 239186df..b0e0733a 100644
--- a/storage-proofs/core/src/parameter_cache.rs
+++ b/storage-proofs/core/src/parameter_cache.rs
@@ -32,7 +32,7 @@ struct LockedFile(File);
 impl LockedFile {
     pub fn open_exclusive_read<P: AsRef<Path>>(p: P) -> io::Result<Self> {
         let f = fs::OpenOptions::new().read(true).open(p)?;
-        f.lock_exclusive()?;
+        // f.lock_exclusive()?;
 
         Ok(LockedFile(f))
     }
@@ -43,7 +43,7 @@ impl LockedFile {
             .write(true)
             .create(true)
             .open(p)?;
-        f.lock_exclusive()?;
+        // f.lock_exclusive()?;
 
         Ok(LockedFile(f))
     }
@@ -73,9 +73,11 @@ impl io::Seek for LockedFile {
 
 impl Drop for LockedFile {
     fn drop(&mut self) {
+      /*
         self.0
             .unlock()
             .unwrap_or_else(|e| panic!("{}: failed to {:?} unlock file safely", e, &self.0));
+            */
     }
 }
 
