--- policy.go.save	2020-07-23 22:49:24.983497550 +0000
+++ policy.go	2020-07-23 22:49:49.166993404 +0000
@@ -83,7 +83,7 @@
 
 // Number of epochs between publishing the precommit and when the challenge for interactive PoRep is drawn
 // used to ensure it is not predictable by miner.
-var PreCommitChallengeDelay = abi.ChainEpoch(150)
+var PreCommitChallengeDelay = abi.ChainEpoch(5)
 
 // Lookback from the current epoch for state view for leader elections.
 const ElectionLookback = abi.ChainEpoch(1) // PARAM_FINISH
